python/private/bzlmod/pip.bzl

```diff
@@ -421,6 +427,12 @@ A dict of labels to wheel names that is typically generated by the whl_modificat
 The labels are JSON config files describing the modifications.
 """,
         ),
+    "override": attr.bool(
```

Comment:
I personally think this is OK since this works similarly to how archive overrides work, but I would like @rickeylev and @groodt to also weigh in here.

This is an obvious case where we may want to have an escape hatch and because it is a root module, we can do this.

However, I am wondering about what happens if a module like this gets published.